Index: src/crypto/cryptlib.h
===================================================================
--- src/crypto/cryptlib.h	(revision )
+++ src/crypto/cryptlib.h	(revision )
@@ -82,7 +82,7 @@
 # ifndef OPENSSL_SYS_VMS
 #  define X509_CERT_AREA          OPENSSLDIR
 #  define X509_CERT_DIR           OPENSSLDIR "/certs"
-#  define X509_CERT_FILE          OPENSSLDIR "/cert.pem"
+#  define X509_CERT_FILE          OPENSSLDIR "/certs/ca-certificates.crt"
 #  define X509_PRIVATE_DIR        OPENSSLDIR "/private"
 # else
 #  define X509_CERT_AREA          "SSLROOT:[000000]"
