Index: src/apps/s_client.c
===================================================================
--- src/apps/s_client.c	(revision )
+++ src/apps/s_client.c	(revision )
@@ -739,7 +739,7 @@
     if (!set_cert_key_stuff(ctx, cert, key))
         goto end;
 
-    if ((!SSL_CTX_load_verify_locations(ctx, CAfile, CApath)) ||
+    if ((!SSL_CTX_load_verify_locations(ctx, CAfile, CApath)) &&
         (!SSL_CTX_set_default_verify_paths(ctx))) {
         /*
          * BIO_printf(bio_err,"error setting default verify locations\n");
