Index: src/apps/s_time.c
===================================================================
--- src/apps/s_time.c	(revision )
+++ src/apps/s_time.c	(revision )
@@ -483,7 +483,7 @@
 
     SSL_load_error_strings();
 
-    if ((!SSL_CTX_load_verify_locations(tm_ctx, CAfile, CApath)) ||
+    if ((!SSL_CTX_load_verify_locations(tm_ctx, CAfile, CApath)) &&
         (!SSL_CTX_set_default_verify_paths(tm_ctx))) {
         /*
          * BIO_printf(bio_err,"error setting default verify locations\n");
